### ISSUE TITLE
How to load scoped grunt plugin?

### ISSUE THREAD

**Issue by dickeylth** [shootaroo/jit-grunt#25]
I've written a grunt plugin under my company's internal npm source, so the grunt plugin' name is: `@abc/grunt-xxx`, as the plugin will be installed under `node_modules/@abc/grunt-xxx` diretory, I've tried custom static mapping in jit-grunt like this:

```
require('jit-grunt')(grunt, {
  'xxx': '@abc/grunt-xxx'
});
```

However it still won't work, the console tells that jit-grunt could not find the plugin:

```
jit-grunt: Plugin for the "xxx" task not found.
If you have installed the plugin already, please setting the static mapping.
See https://github.com/shootaroo/jit-grunt#static-mappings
```

So how should I config the plugin? Thank you!

**Comment by shootaroo** [comment]
jit-grunt does not yet support the private repository.

Please for the time being to avoid the following:
```js
  require('jit-grunt')(grunt, {
    xxx: 'node_modules/@abc/grunt-xxx/tasks/xxx.js'
  });
```
or
```js
  grunt.loadNpmTasks('@abc/grunt-xxx');
```
I will fix as soon.

**Comment by dickeylth** [comment]
OK, thank you for your reply! Waiting for your update!

**Comment by shootaroo** [comment]
I released v0.9.1. Use it.

**Comment by dickeylth** [comment]
OK, it works, thank you for your quick response!
